### PR TITLE
Adds a sound effect for hypospray/medipen injections

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

Adds a sound effect for hypospray/medipen injections. Doesn't affect sleepy pens or HMS injectors

Sound is sourced from the open source CM codebase.

## Why It's Good For The Game

> Adds a sound effect for hypospray/medipen injections. Doesn't affect sleepy pens or HMS injectors

Satisfactory, visceral feedback of injection. Sounds nice. Makes stealth injections less noticeable, I guess, not that there's any good candidates for that with medipens.

## Changelog

:cl:
sound: Adds a sound effect for hypospray/medipen injections. Doesn't affect sleepy pens or HMS injectors
/:cl:

### CODE REVIEW

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 1]
```diff
@@ -40,6 +40,8 @@
 	if(reagents.total_volume && (ignore_flags || affected_mob.try_inject(user, injection_flags = INJECT_TRY_SHOW_ERROR_MESSAGE))) // Ignore flag should be checked first or there will be an error message.
 		to_chat(affected_mob, span_warning("You feel a tiny prick!"))
 		to_chat(user, span_notice("You inject [affected_mob] with [src]."))
+		if(!stealthy)
+			playsound(get_turf(affected_mob), 'sound/items/hypospray.ogg', 50, TRUE)
```
Comment (MrMelbert) [inline code/modules/reagents/reagent_containers/hypospray.dm:44]:
```suggestion
			playsound(affected_mob, 'sound/items/hypospray.ogg', 50, TRUE)
```
it don't matter

**Comment by Helg2** [comment]
Can you give a sound example?

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 2]
```diff
@@ -119,6 +121,8 @@
 	list_reagents = list(/datum/reagent/medicine/epinephrine = 10, /datum/reagent/toxin/formaldehyde = 3, /datum/reagent/medicine/coagulant = 2)
 	custom_price = PAYCHECK_CREW
 	custom_premium_price = PAYCHECK_COMMAND
+	/// If TRUE, won't play a noise when injecting.
+	var/stealthy = FALSE
```
Comment (Helg2) [inline code/modules/reagents/reagent_containers/hypospray.dm:125]: Just add the var to hypospray not medipen.

**Comment by moocowswag** [comment]
its lowkey a balance change but a welcome one.

**Comment by MrMelbert** [comment]
Do you have credit for the source of this sound effect? I know I've heard it on other servers before

**Comment by DrDiasyl** [comment]
> Do you have credit for the source of this sound effect? I know I've heard it on other servers before

This sound is used at Colonial Marines.

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]
Source of sound needs to be present in PR description for traceability and to verify any licensing issues, otherwise good to go

**Comment by carlarctg** [comment]
> > Do you have credit for the source of this sound effect? I know I've heard it on other servers before
> 
> This sound is used at Colonial Marines.

Somehow I forgot to say that, but yeah, that's where it's from, same with the health analyzer noise I PRed a bit ago.

**Comment by carlarctg** [comment]
@ExcessiveUseOfCobblestone

**Comment by Ghommie** [comment]
Umh, the sound is used on several other servers and I don't see it in any attributions text file so I can only suppose this is yet another of those ok to use sounds. 🤷

**Pull request merged**